script/Arcueid/Day 1/01_00_ARC01_5.txt

```diff
@@ -60,7 +60,7 @@ After a few seconds of waiting,
 [sha:852b3a0cb9a7bf40a5ec78cb42c431a362e34696]{
 -- Page 247, Offset 1795.
 -- 「……ここのドア、全自動になったんだ。昔は守衛さんがいたんだけどな……」
-"...The door is fully automatic now. There used to be a guard..."
+"So the door's fully automatic now... There used to be a guard."
```

Comment:
guard before?

Reply:
Not really necessary with `used to` already there